### DISCUSSION TITLE
Search/filter does nothing on Mac

### DISCUSSION THREAD

**Discussion by basscl** [iterate-ch/cyberduck#15446] [Q&A]
When I try to use Search function (also called Filter in the documentation), either by using the shortcut (Apple+\) or Go>Search, absolutely nothing happens. No popup/modal, no place to type anything in to search for. This is the case whether in Bookmarks, History, or Browser view. Am I doing something wrong? I really need this functionality...

I'm running Cyberduck 8.7.1 on macOS Monterey 12.0.1

**Comment by dkocher** [comment]
The _Search_ toolbar item should gain focus when choosing _Go → Search_. You may need to add the _Search_ toolbar item by adding _View → Customize Toolbar…_.

![Screenshot 2023-12-09 at 12 03 26](https://github.com/iterate-ch/cyberduck/assets/71733/a40a0f71-4ba2-4e50-849e-fa231500593b)
![Screenshot 2023-12-09 at 12 03 11](https://github.com/iterate-ch/cyberduck/assets/71733/f98264dd-c3ab-4dad-8097-de2ca3f050d4)
